src/app/modules/shared/components/input/select/select.component.html

```diff
@@ -1,60 +1,29 @@
-<ng-container *ngIf="isApp; else popover">
-  <ion-label color="medium" position="stacked">
-    {{ label() | translate }}
-  </ion-label>
-  <ion-button
-    class="open-actionsheet"
-    role="combobox"
-    aria-expanded="false"
-    aria-haspopup="dialog"
-    expand="full"
-    type="button"
-    fill="clear"
-    (click)="openSelect()"
-  >
-  </ion-button>
-  <ion-icon class="value-icon" *ngIf="valueIcon" [name]="valueIcon"></ion-icon>
-  <ion-text>
-    {{ valueText() | translate }}
-  </ion-text>
+<ion-button
+  class="open-actionsheet"
+  role="combobox"
+  aria-expanded="false"
+  aria-haspopup="dialog"
+  expand="full"
+  type="button"
+  fill="clear"
+  (click)="openSelect()"
+>
+</ion-button>
+<div class="select">
+  <div class="select__inner">
+    <ion-label color="medium" position="stacked">
+      {{ label() | translate }}
+    </ion-label>
+    <div class="select__icon-text">
+      @if (valueIcon()) {
+        <ion-icon class="value-icon" [src]="valueIcon()"></ion-icon>
```

Comment:
ser ut at jeg fortsatt kan bruke [name] for våre egne ikoner i ion-icon istedet for [src]